archive_feedstock.py

```diff
@@ -52,13 +53,24 @@ def feedstocks(directory="archive"):
                 line = line.strip()
                 if line and not line.startswith("#"):
                     yield line
+        if not check_only:
```

Comment:
Oops, yep I missed this part :D 

With the current exception accumulation we have now, if an error occurs and the repo cannot be archived, we will have deleted the files and committed in the temporarily cloned repo. I guess that's not very problematic in CI usage, but ideally we would only commit on full success.

I don't think it's really important though and if it works most of the time, it should be ok. Just writing it down in case that edge case was not considered.

Reply:
Oops, the rest of the conversation hadn't loaded for some reason and I am now realizing you did talk about this. Sorry for the noise 😬